Coords.cs

```diff
@@ -0,0 +1 @@
+record Coord(int x, int y) {}
```

Comment:
I don't like the name, use the full name. Also, make it `record struct` to train with it as well.
Consider using `char letter, int number` as parameters